include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -556,6 +571,8 @@ namespace RINGMesh
         };
         GeoModelBuilderTSolid( GeoModel3D& geomodel, std::string filename );
 
+        virtual ~GeoModelBuilderTSolid() = default;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this "virtual" specifier or replace by "override". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3576)